prime-dotnet-webapi/Models/Logging/VendorApiLog.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+    [Table("VendorApiLog")]
+    public class VendorApiLog : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string UserId { get; set; }
```

Comment:
Can this be named `ServiceAccountUsername` instead?  See Filip's email Wed 2023-05-03 9:55 AM

Reply:
Column name is updated.